codalab/bundles/named_bundle.py

```diff
@@ -18,7 +18,7 @@ class NamedBundle(Bundle):
     # Don't format specs
     # fmt: off
     METADATA_SPECS = [
-        MetadataSpec('name', str, 'Short variable name (not necessarily unique); must conform to %s.' % spec_util.NAME_REGEX.pattern, short_key='n',),
+        MetadataSpec('name', str, 'Bundle names can only contain letters, digits, underscores, dots and dashes. Names can only start with a letter or an underscore.', short_key='n'),
```

Comment:
How about: Short name (not necessarily unique), which must start with a letter or underscore and can only contain letters, digits, underscores, periods, and dashes".